### ISSUE TITLE
Error Message While CTRL-Clicking

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3451]
In chasing through some code in LeoPyRef, I CTRL-Clicked on the string _safeMoveToThreadNext_.  Leo did navigate to its definition, but the following error message was printed to the console:
```
Uncaught exception in Leo...
Traceback (most recent call last):
  File "C:\Users\tom\AppData\Local\Programs\Python\Python311\Lib\code.py", line 90, in runcode
    exec(code, self.locals)
  File "<input>", line 1, in <module>
AttributeError: 'str' object has no attribute 'p'
```
More tests showed the same error for each CTRL-Click I tried, and often the error message was emitted several times.  In every case the navigation completed successfully.
```
Leo 6.7.4-devel, devel branch, build 947ea935f5
2023-07-06 08:01:34 -0500
Python 3.11.4, PyQt version 6.4.3
Windows 10 AMD64 (build 10.0.19045) SP0
```

**Comment by tbpassin** [comment]
I'm also seeing the message when I insert or delete a node.

**Comment by edreamleo** [comment]
@tbpassin This looks like an installation problem. Everything here works as expected.

**Comment by tbpassin** [comment]
It must have been some anomaly, maybe some corrupted bitcode.  I repeated these operations on a Linux VM, and also with a different version of Python.  Then I restarted a new Leo session in Windows with the original Python exe, 3.11.4.  After this the error messages did not get emitted.

**Comment by edreamleo** [comment]
@tbpassin I'm wondering why you have such installation problems. Do you have any idea?

**Comment by tbpassin** [comment]
I get more crashes than I would like.  Some of them are when Leo is running, and the Windows error code usually claims some kind of memory problem in a Python DLL, like a buffer overrun.  Sometimes it is claimed to be in a Qt DLL. A few times, Windows crashes.  A few times  a memory diagnostic claims there is a memory problem, other times not.

Once I kept getting Leo/Python crashes at the exact same location. After I rebooted, that stopped.  Also, most of my problems happen in Leo/Python, not other long-running programs like the browser or email client.

So I may have a problem with RAM, but so far it's too intermittent to diagnose reliably.  A year ago I took it to Best Buy and asked them to replace all the memory.  But their memory diagnostic didn't find any errors, and apparently the right memory chips were hard to come by so they would only have been able to replace one of the two banks.  I didn't see much point in that.  So nothing was changed.  Grrrr.
